### ISSUE TITLE
feat: add new linter check for no hardcoded strings where translatable strings should be

### ISSUE THREAD

**Issue by matthewkurtti** [ourjapanlife/findadoc-web#1407]
# Problem
In our [docs](https://documentation.findadoc.jp/development/start-coding/localization), we explain why we use translatable strings instead of hard coded strings. However, there currently is no linter check to make sure this is happening. Let's fix that

(Only text that would be displayed to users needs to use translatable strings. Other text, like comments in the code, can be hard coded)

# Success

The linter check is able to give back an error if new changes uses hard coded strings instead of translatable strings, and does not give back an error if changes use translatable strings correctly

### Requirements

- [ ] Linter check gives back an error if hard coded strings are used
- [ ] Linter check passes if translatable strings are used

# Implementation notes
